### ISSUE TITLE
Dark mode

### ISSUE THREAD

**Issue by Yodzorah** [jlord/git-it-electron#302]
An ability to switch app's theme to dark is desperately needed. Because there's only Light mode available, I'll have to find another git tutorial. It's _that_ critical.

**Comment by jim3** [comment]
Yes please, a dark mode. My eyes cannot stand light mode...ever.

**Comment by Rom1deTroyes** [comment]
Any news on this issue ?

Adding a Material UI Dark theme in a Electron App don't seems to be a complex task, and my eyes are already bleeding after the first Challenge 👁️‍🗨️ 👁️

**Comment by g-h-97** [comment]
Was going to open a similar issue, sorry but this must be top priority imho,

**Comment by g-h-97** [comment]
@bzbug, @jim3, @Rom1deTroyes use this workaround to apply dark mode for now,

- Install `Dark Reader` Chrome/Brave extension 
- Go to `chrome://extensions` find Dark Reader, click Allow access to file URLs
- Open Git-it `index.html` on the web browser directly, E.g path on linux `Git-it-Linux-x64/resources/app/built/en-US/challenges/repository.html`

The only drawback with this method is that Git-it can't validate your progress, Hopefully the devs will implement dark mode soon,
